src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import core.basesyntax.utils.DataUtils;
+
 public class SalaryInfo {
+    static final String FORMAT = "dd.MM.yyyy";
+    private final StringBuilder report = new StringBuilder();
+    private final DataUtils dataUtils = new DataUtils();
+    private int salaryOfEmployee;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String [] splitData;
+        for (String name : names) {
+            for (int i = 0; i < data.length; i++) {
+                if (data[i].contains(name)) {
+                    splitData = data[i].split(" ");
+                    String reportedDate = splitData[0];
+                    int reportedHours = Integer.parseInt(splitData[2]);
+                    int hourlyWage = Integer.parseInt(splitData[3]);
```

Comment:
0, 2, 3 - magic numbers
all indexes should be constant
```suggestion
                    splitData = data[i].split(" ");
                    String reportedDate = splitData[0];
                    int reportedHours = Integer.parseInt(splitData[2]);
                    int hourlyWage = Integer.parseInt(splitData[3]);
```

Reply:
done